### PR TITLE
update template to 3.x api

### PR DESCRIPTION
**Pull request by luipir**
## Description
Fix some inconsistencies of processing script template

fix promoted inside GeoMove [1] project for Cartolab [2] (A Coruña university)

[1] http://cartolab.udc.es/geomove/
[2] http://cartolab.udc.es

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

- [x] Commit messages are descriptive and explain the rationale for changes
- [ ] Commits which fix bugs include `fixes #11111` in the commit message next to the description
- [ ] Commits which add new features are tagged with `[FEATURE]` in the commit message
- [ ] Commits which change the UI or existing user workflows are tagged with `[needs-docs]` in the commit message and contain sufficient information in the commit message to be documented
- [x] I have read the [QGIS Coding Standards](https://docs.qgis.org/testing/en/docs/developers_guide/codingstandards.html) and this PR complies with them
- [ ] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [ ] New unit tests have been added for core changes
- [x] I have run [the `scripts/prepare-commit.sh` script](https://github.com/qgis/QGIS/blob/master/.github/CONTRIBUTE.md#contributing-to-qgis) before each commit

### CODE REVIEW

File: python/plugins/processing/script/ScriptTemplate.py [hunk 1]
```diff
@@ -150,7 +150,7 @@ def processAlgorithm(self, parameters, context, feedback):
         )
 
         # Send some information to the user
-        feedback.pushInfo('CRS is {}'.format(source.sourceCrs().authid()))
+        feedback.pushInfo('CRS is {}'.format(source.crs().authid()))
```
Comment (nyalldawson) [inline python/plugins/processing/script/ScriptTemplate.py]: This one is already correct - QgsFeatureSource has sourceCrs, not crs().

**Comment by nyalldawson** [comment]
@luipir some of this is still valid - the parameters name needs to be changed, like you've done here

**Comment by luipir** [comment]
@nyalldawson sure sorry...

**Comment by nyalldawson** [comment]
Thanks!

**Review by nyalldawson — APPROVED** [review]

**Pull request merged**